docs/guides/expectations/advanced/how-to-create-expectations-that-span-multiple-batches-using-evaluation-parameters.md

```diff
@@ -2,4 +2,147 @@
+
+    When executed in the notebook, this Expectation will generate an [Expectation Validation Result](../../../reference/validation). Most values will be missing, since interactive evaluation was disabled.
+
+    ```python
+        {
+          "result": {},
+          "success": null,
+          "meta": {},
+          "exception_info": {
+            "raised_exception": false,
+            "exception_traceback": null,
+            "exception_message": null
+          }
+        }
+    ```
+
+   :::warning
+
+   Your URN must be exactly correct in order to work in production. Unfortunately, successful execution at this stage does not guarantee that the URN is specified correctly and that the intended parameters will be available when executed later.
+
+   :::
+
+5. **Save your Expectation Suite**
+
+    ```python
+    downstream_validator.save_expectation_suite(discard_failed_expectations=False)
+    ```
+
+    This step is necessary because your ``$PARAMETER`` will only function properly when invoked within a Validation operation with multiple Validators. The simplest way to execute such an operation is through a :ref:`Validation Operator <reference__core_concepts__validation__validation_operator>`, and Validation Operators are configured to load Expectation Suites from Expectation Stores, not memory.
+
+6. **Execute an existing Validation Operator on your upstream and downstream Validators.**
+
+    You can do this within your notebook by running ``context.run_validation_operator``.
+
+    ```python
+    results = context.run_validation_operator(
+        "action_list_operator",
+        assets_to_validate=[
+            upstream_validator,
+            downstream_validator
+        ]
+    )
+    ```
+
+7. **Rebuild Data Docs and review results in docs.**
+
+    You can do this within your notebook by running:
+
+    ```python
+    context.build_data_docs()
+    ```
+
+    You can also execute from the command line with:
+
+    ```bash
+    great_expectations docs build
+    ```
+
+    Once your Docs rebuild, open them in a browser and navigate to the page for the new Validation Result.
+
+    If your Evaluation Parameter was executed successfully, you'll see something like this:
+
+    ![image](../../../../docs/images/evaluation_parameter_success.png)
+
+    If it encountered an error, you'll see something like this. The most common problem is a mis-specified URN name.
+
+    ![image](../../../../docs/images/evaluation_parameter_error.png)
+
+   :::warning
+
+   In general, the development loop for testing and debugging URN and Evaluation Parameters is not very user-friendly. We plan to simplify this workflow in the future. In the meantime, we welcome questions in the `Great Expectations discussion forum <https://discuss.great_expectations.io>`_ and `Slack channel <https://great_expectations.io/slack>`_.
```

Comment:
```suggestion
   In general, the development loop for testing and debugging URN and Evaluation Parameters is not very user-friendly. We plan to simplify this workflow in the future. In the meantime, we welcome questions in the [Great Expectations discussion forum](https://discuss.great_expectations.io) and [Slack channel](https://great_expectations.io/slack).
```

Two links are still RTD format.

Reply:
Will fix -- thanks for the catch; I guess that one got away.